src/wmtk/operations/OptimizationSmoothing.cpp

```diff
@@ -140,17 +154,27 @@ void OptimizationSmoothing::create_solver()
 
 std::vector<simplex::Simplex> OptimizationSmoothing::execute(const simplex::Simplex& simplex)
 {
-    WMTKProblem problem(
-        mesh().create_accessor(m_energy->attribute_handle().as<double>()),
-        simplex,
-        m_invariants,
-        *m_energy);
-
-    auto x = problem.initial_value();
-    try {
-        m_solver->minimize(problem, x);
-    } catch (const std::exception&) {
-        return {};
+    std::vector<simplex::Simplex> simplices_to_smooth;
```

Comment:
It's not clear to be whether the multi mesh behavior or the optimization should be to trigger multiple optimizations. This should be a user driven choice